exercises/matching-brackets/canonical-data.json

```diff
@@ -118,6 +118,18 @@
       },
       "expected": false
     },
+    {
+      "uuid": "1d5c093f-fc84-41fb-8c2a-e052f9581602",
+      "description": "paired and wrong deep nested brackets",
```

Comment:
I'm concerned that now that this is a new test case instead of reimplementing, the description, when contrasted with the existing "paired and wrong nested brackets", is confusing. The level of nesting is equal to the existing one, so the word "deep" sounds wrong to me.

My preference is that the existing case be reimplemented for reasons I've already discussed previously, but if this must be added as a new case, I would ask for a more indicative name. My instinct is "paired and wrong nested brackets but innermost are correct"

Reply:
I would rather adopt your suggestion, but keep the new test.

For now, I tend to agree with the argument that we want a more indicative test in case of a wrong implementation.

We established that are multiple ways of messing up "paired but wrong order". So instead of one big input value that solves multiple edge cases, having a specific test for each one would be, maybe, more indicative of what is wrong with their implementation.

The new test can be added only when and if a student brings it up here.

I went back and forth with this in my mind, but ended up settling with your previous argument, but now hearing you're team reimplementation, I'm confused hahaha